.eslintrc.js

```diff
@@ -49,5 +49,6 @@ module.exports = {
         format: ["camelCase", "PascalCase", "UPPER_CASE"],
       },
     ],
+    "@typescript-eslint/ban-ts-comment": ["off"],
```

Comment:
I remember you saying this line didn't end up allowing the ts-ignore line. Is that still the case?

Reply:
Yes but it's still needed to allow the `ts-nocheck` which I am using